Bai2/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
+*.json
```

Comment:
why do you need to ignore the json file?

Reply:
i don't wanna push folder node_modules to github because it's so heavy

Reply:
No, this *.json

Reply:
because I got warnings
warning: in the working copy of 'package-lock.json', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'package.json', LF will be replaced by CRLF the next time Git touches it
therefore i find it unnecessary to push